packages/tuffex/packages/components/src/switch/style/index.scss

```
@@ -21,34 +24,63 @@
+      border-top-color: transparent;
+      opacity: 0;
+      transition: opacity 0.25s ease-in-out;
+    }
   }
 
   &.is-active {
     background-color: var(--tx-color-primary, #409eff);
 
     .tuff-switch__thumb {
       left: 50%;
+      // Brightens the ring stroke along with the thumb; `transparent` keeps its
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)